src/Message/AbstractPart.php

```diff
@@ -240,7 +240,7 @@ final public function getDecodedContent(): string
     /**
      * Get raw message content.
      */
-    final protected function doGetContent(string $partNumber): string
+    final protected function doGetContent(string $partNumber): ?string
```

Comment:
@roy-bongers  Dit fixed de return type error, maar is dit geen issue in de imap2 library die `null` teruggeeft maar eigenlijk bijvoorbeeld `false` of `""` terug moet geven?

Reply:
Tja, zeg 't maar. Hij komt hier in terecht of in de php-imap (library) imap_fetchbody functie. Die heb ik lokaal echter niet draaien. php.net zegt dat deze string|false returned: https://www.php.net/manual/en/function.imap-fetchbody.php

![Screenshot 2023-02-03 at 09 51 57](https://user-images.githubusercontent.com/10220797/216554848-8cc16ec1-103e-47ae-acbb-f9d303d6d181.png)